src/coprocessor/dag/builder.rs

```diff
@@ -150,61 +152,78 @@ impl DAGBuilder {
         for mut ed in executor_descriptors {
             summary_slot_index += 1;
 
-            let new_executor: Box<dyn BatchExecutor> = match ed.get_tp() {
-                ExecType::TypeSelection => {
-                    COPR_EXECUTOR_COUNT
-                        .with_label_values(&["batch_selection"])
-                        .inc();
-
-                    Box::new(
-                        BatchSelectionExecutor::new(
-                            config.clone(),
-                            executor,
-                            ed.take_selection().take_conditions().into_vec(),
-                        )?
-                        .with_summary_collector(C::new(summary_slot_index)),
-                    )
-                }
-                ExecType::TypeAggregation | ExecType::TypeStreamAgg
-                    if ed.get_aggregation().get_group_by().is_empty() =>
-                {
-                    COPR_EXECUTOR_COUNT
-                        .with_label_values(&["batch_simple_aggr"])
-                        .inc();
+            let new_executor: Box<dyn BatchExecutor<StorageStats = S::Statistics>> =
```

Comment:
Quite annoying diff caused by reformatting :(

Reply:
Ues `Split` mode to review this file will make your life better.